### PR TITLE
Allow for multiple mock handler setup in single call

### PR DESCRIPTION
**Pull request by koorgoo**
PR for #188. I like suggested format. And, honestly, I want to speed up the release 2.2 😄

### CODE REVIEW

**Comment by koorgoo** [comment]
Need to add description in README...

**Comment by jakerella** [comment]
Looks good so far! Yep, documentation is necessary, I would also add a test for an empty array and a failure if one (but not all) of the array entries is bad - in other words, what would be returned?

File: README.md [hunk 1]
```diff
@@ -312,7 +313,18 @@ $.mockjax({
 });
 ```
 
-### Define a Response ###
+### Defining Multiple Requests ###
+
+Since version 2.2 it is allowed to define several requests at once.
```
Comment (jakerella) [inline README.md]: Be sure to document what is returned!

**Comment by jakerella** [comment]
I hear you about making 2.2.0 come sooner, thanks! Please also add a longer description in the PR on how to use this new feature.

**Comment by koorgoo** [comment]
> test for ... a failure if one (but not all) of the array entries is bad

Since `$.mockjax([...])` calls `$.mockjax({})` I think to test bad entries will be better done in tests for `$.mockjax({})`. By the way I saw how handlers are registered - new handler registering does not check whether settings argument is `null`, `undefined`, or bad object, settings are just pushed to `mockHandlers` array. On the other hand, test for empty settings array must exist.
What do you think?

**Comment by koorgoo** [comment]
Also I thought about `$.mockjax.clearAll(handlers)` would be a good addition to multiple mock handler setup. Your thoughts?

```javascript

var handlers = $.mockjax([
  ...
]);

$.mockjax.clearAll(handlers);
```

**Comment by jakerella** [comment]
I see what you're saying about the handling, all I want to check is that the functionality for what is returned is not changed by the fact that we are using an array. This also just goes to having proper tests in case we change how things are returned in the future.

For the `clearAll` idea, sounds fine, but that is a separate feature! Please add an issue if you'd like to see it. :)

For now I see a few things:

 - [ ] Document what is returned in the readme
 - [ ] Add a test for other, non-object entries in the array
 - [ ] Add a test for an empty array

Thank you!

**Comment by koorgoo** [comment]
Done.
